introduction.md

```diff
@@ -1 +1,8 @@
 I am Diego and my favorite Code School path is JavaScript
+
+# Deadly Skills: 
+* C#
+* JavaScript
+* Git
+
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?